app/models/spree/ability.rb

```diff
@@ -29,6 +29,7 @@ def initialize(user)
         can :update, Order do |order, token|
           order.user == user || order.token && token == order.token
         end
+        can [:index], :invoice
```

Comment:
The only invoices controller is `Spree::Admin::InvoicesController`, and I think this rule applies for all users, which doesn't seem right to me. Should it also be moved to `add_order_management_abilities`?

Reply:
This will not change anything on the final result, because we're checking the permission on the action's method level.
but you're right, it'll make more sense to grant this permission from `add_order_management_abilities`.